src/app.rs

```diff
@@ -21,24 +21,106 @@
 //! - UDP trackers: the user can enable multiple UDP tracker on several ports.
 //! - HTTP trackers: the user can enable multiple HTTP tracker on several ports.
 //! - Tracker REST API: the tracker API can be enabled/disabled.
+use std::collections::HashMap;
+use std::net::SocketAddr;
 use std::sync::Arc;
 
+use derive_more::Constructor;
 use log::warn;
+use tokio::sync::Mutex;
 use tokio::task::JoinHandle;
 use torrust_tracker_configuration::Configuration;
 
 use crate::bootstrap::jobs::{health_check_api, http_tracker, torrent_cleanup, tracker_apis, udp_tracker};
 use crate::{core, servers};
 
+/// Returned by the service check.
+#[derive(Debug, Constructor)]
+pub struct CheckJob {
+    pub binding: SocketAddr,
+    pub info: String,
+    pub job: JoinHandle<Result<String, String>>,
+}
```

Comment:
hi @da2ce7 rename to `HealthCheckResult`?

Reply:
Done :smile: